src/components/general/Slider/styles.less

```diff
@@ -45,7 +45,6 @@
         top: 50%;
         transform: translateY(-50%);
         z-index: 2;
-        background: var(--color-primary);

```

Comment:
Should probably apply similar changes to the markers aswell. Markers to the right of the dot are "highlighted"

Reply:
Remove style and the logic for applying it or just the style?

Reply:
Just style

Reply:
Removed style